src/contrib/cluster/Akka.Cluster.Sharding/Serialization/ClusterShardingMessageSerializer.cs

```diff
@@ -63,15 +65,19 @@ public class ClusterShardingMessageSerializer : SerializerWithStringManifest
         private const string ShardStatsManifest = "DB";
         private const string GetShardRegionStatsManifest = "DC";
         private const string ShardRegionStatsManifest = "DD";
+        private const string GetClusterShardingStatsManifest = "GS"; // This is "DE" in JVM
+        private const string ClusterShardingStatsManifest = "CS"; // This is "DF" in JVM
         private const string GetCurrentRegionsManifest = "DG";
         private const string CurrentRegionsManifest = "DH";
 
+        private const string GetCurrentShardStateManifest = "FA";
+        private const string CurrentShardStateManifest = "FB";
         private const string GetShardRegionStateManifest = "FC";
         private const string ShardStateManifest = "FD";
         private const string CurrentShardRegionStateManifest = "FE";
-        
-        private const string GetClusterShardingStatsManifest = "GS"; // This is "DE" in JVM
-        private const string ClusterShardingStatsManifest = "CS"; // This is "DF" in JVM
+
+        private const string EventSourcedRememberShardsMigrationMarkerManifest = "SM";
+        private const string EventSourcedRememberShardsState = "SS";
 
         #endregion
```

Comment:
All of the old manifests are intact - we've only added new ones. So far, so good as far as the wire format goes.